test/index.spec.js

```diff
@@ -209,6 +219,8 @@ describe('hermes', function () {
     });
 
     it('should unsubscribe workers from rabbitmq (specific workers in queue)', function (done) {
+      Events.prototype.createQueues.yields();
+      Events.prototype.createExchanges.yields();
```

Comment:
even though they always yield I still want to specify in which test they should yield. if for example I am testing an error case, this should not be called and if it was the test would fail. however if we always yielded I might miss the fact that this function gets called.